rackcity/views/asset_views.py

```diff
@@ -494,6 +491,16 @@ def asset_delete(request):
             {"failure_message": Status.AUTH_ERROR + str(auth_error)},
             status=HTTPStatus.UNAUTHORIZED,
         )
+    if existing_asset.model.is_blade_chassis():
+        blades = Asset.objects.filter(chassis=existing_asset)
+        if len(blades)>0:
```

Comment:
nit:  you could just do `if Asset.objects.filter(chassis=existing_asset).exists():`